lib/auth/login_log.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Auth.LoginLog do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias Auth.Repo
+
+  schema "login_logs" do
+    field :email, :string
+    field :ip_address, :string
+    field :auth_provider, :string
+    belongs_to :person, Auth.Person
+    belongs_to :user_agent, Auth.UserAgent
```

Comment:
The `login_attempt` should _definitely_ not "belong" to a `:user_agent`.
Storing the UserAgent in a separate table is just a way to normalise the data.
Do we need to have an Ecto association for that?